onboarding/forms.py

```diff
@@ -73,6 +73,10 @@ def clean(self):
         return cleaned_data
 
 
+class NycAddressForm(AptNumberWithConfirmationForm, AddressAndBoroughFormMixin):
+    pass
```

Comment:
Note that this isn't a `ModelForm`. There's just too many wacky edge cases that are hard to deal with if we make it one: for instance, because a `ModelForm` calls the underlying instance's `full_clean()`, it will raise validation errors the user can't do anything about if they change their address from one that is outside NYC to one that is inside NYC.  So we'll just use this for validation of the user's address/apt # but actually save it to the model in the associated GraphQL mutation.